pkg/client/USAGE.md

```
@@ -18,11 +18,11 @@ func main() {
 	)
 
 	ctx := context.Background()
-	res, err := s.Ledger.GetInfo(ctx)
+	res, err := s.Ledger.V1.GetInfo(ctx)
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)